lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,9 +586,10 @@ module.exports = class MethodMappings {
 
         async sendKeysToElement(webElementOrId, value) {
           if (Array.isArray(value)) {
+            if (value.includes(undefined) || value.includes(null)){
+              throw new Error('sendKeys - each key must be a number or string');
+            }
             value = value.join('');
-          } else {
-            value = String(value);
```

Comment:
Any particular reason that we had removed `else`?

Reply:
Yes, because this function [sendKeys](https://github.com/SeleniumHQ/selenium/blob/468b5e345d430688776aa64f65fc2a1395e7aee5/javascript/node/selenium-webdriver/lib/webdriver.js#L2521) only expects string or a number. If value is a number then else condition will converts it to string which is wrong. The same thing can happen in the case of undefined and null. Also if the value is a string then else case is meaningless.